backend/lib/calculate/adequacy.py

```diff
@@ -38,7 +38,7 @@ def _find_closest_location(point, locations, measurer, exit_distance_in_meters=N
     provider = {
         'id': point['id'],
         # FIXME - Return real lists of closest providers.
-        'closest_providers': [1],
+        'closest_provider_point': closest_provider,
```

Comment:
Maybe let's rename this `closest_point`? We call it `min_point` in `base.py`

Reply:
Should we actually transfer to the frontend?